test/e2e/backups/deletion.go

```diff
@@ -178,6 +178,11 @@ func runBackupDeletionTests(client TestClient, veleroCfg VeleroConfig, backupNam
 			return errors.Wrap(err, "exceed waiting for snapshot created in cloud")
 		}
 	}
+
+	// Hit issue: https://docs.aws.amazon.com/AWSEC2/latest/APIReference/errors-overview.html#:~:text=SnapshotCreationPerVolumeRateExceeded
+	// Sleep for more than 15 seconds to avoid this issue.
+	time.Sleep(1 * time.Minute)
```

Comment:
Do you mean that we should try to perform backup (with taking snapshot) multi-times until get a success one? The problem is taking snapshot frequently in short time will trigger SnapshotCreationPerVolumeRateExceeded, so wait should be a best way, otherwise there will be no successful snapshot all the time. Or do I misunderstand your suggestion?

Reply:
I understand your point generally, but for this specific case, no condition should be check according to description for SnapshotCreationPerVolumeRateExceeded in AWS document.

Reply:
It's weird how sleep is added after the delete snapshot code and it's supposed to help with creation rate limits.

Reply:
Maybe something in gingko AfterEach would be more fitting if it's meant to throttle when the next test starts

Reply:
@kaovilai   Yes, we can split this test script into 2 tests with different workload (A,B) according to test purpose as below, then no need to avoid snapshot throttle issue, but adding a 1 minute sleep is acceptable for now with using the same workload (A), if there're more tests on this deletion scenario, we can consider split them into different tests.

> Create backup for workload A
> Delete the backup and verify it's gone
> Create another backup for workload A
> Remove the content from S3 and double check the backup CR is gone.